### ISSUE TITLE
Bug: activeIndex not working if using 3 items on screen

### ISSUE THREAD

**Issue by nohaapav** [maxmarinich/react-alice-carousel#162]
I'm using 3 elements on screen `:responsive {0 {:items 3}}` with 9 items, however if i set activeIndex to the last element (8) and print result of the `onInitialized` i got following:

```
 #js {:item 3, :slide 1, :itemsInSlide 3, :isNextSlideDisabled false, :isPrevSlideDisabled false}
```
Clearly not correct position. Basically i see slides on indexes [3, 4, 5]

What i'm trying to achieve is something like cover carousel / scaling up carousel see:

Has `previous` & `next`:
<img width="1006" alt="Screen Shot 2021-01-11 at 3 50 56 PM" src="https://user-images.githubusercontent.com/4179104/104197157-f4f22d80-5424-11eb-9c81-1192ba13ccb5.png">

No `previous` items:
<img width="1033" alt="Screen Shot 2021-01-11 at 3 51 27 PM" src="https://user-images.githubusercontent.com/4179104/104197209-063b3a00-5425-11eb-9325-84c21e9c2996.png">

No `next` items:
<img width="1017" alt="Screen Shot 2021-01-11 at 3 51 12 PM" src="https://user-images.githubusercontent.com/4179104/104197255-16531980-5425-11eb-8056-4b778b6e544f.png">

Item in the middle is **selected** one.

Does anyone tried to achieve similar goal ?

**Comment by nohaapav** [comment]
Basically i can set blank space (no previous/next) by adding additional items in the beginning and end of items. I can't figure out how to fix positioning although.

**Comment by maxmarinich** [comment]
Fixed in #163. Available from v2.2.1.
